### PR TITLE
DB Connectivity established

### PR DESCRIPTION
**Pull request by prafullmaru**
As a part of this change, drafted basic version to connect API with database and return data for API call

image

### CODE REVIEW

File: ePanchayat.API/CommonUtils/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace ePanchayat.API.CommonUtils
+{
+    public struct Constants
+    {
+        public const string ConnectionString = @"Data Source =MARU\MARU; Initial Catalog =ePanchayat;User ID = pmaru; Password=pmaru@123;";
```
Comment (mohandivraniya) [inline ePanchayat.API/CommonUtils/Constants.cs:5]: You should never has username and password in connection string added like this. we can move it to a secure location and read from there.

File: ePanchayat.API/Repository/PanchayatRepository.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        public List<Panchayat> GetPanchayats()
+        {
+
+            var dataset = _sqlDataAccess.GetDataSetByStoredProc("PanchayatGet_sp");
+            return MapToPanchayat(dataset);
+        }
+
+        private List<Panchayat> MapToPanchayat(DataSet dataset)
+        {
+            var panchayats = dataset.Tables[0].AsEnumerable().Select(row => new Panchayat()
```
Comment (mohandivraniya) [inline ePanchayat.API/Repository/PanchayatRepository.cs:25]: Should not we check null before attempting to get table by index?

File: ePanchayat.API/CommonUtils/Constants.cs (hunk 1 above)
Comment (prafullmaru) [reply ePanchayat.API/CommonUtils/Constants.cs:5]: With initial set-up we are okay for now, once set-up is completed we will address such vulnerability.

**Review by mohandivraniya — APPROVED** [review]
Approved

**Pull request merged**